bedrock/firefox/views.py

```diff
@@ -594,7 +596,10 @@ def get_template_names(self):
             template = "firefox/nightly/whatsnew.html"
         elif channel == "developer":
             if show_57_dev_whatsnew(version):
-                template = "firefox/developer/whatsnew.html"
+                if switch("firefox-developer-whatsnew-mdnplus") and ftl_file_is_active("firefox/whatsnew/whatsnew-developer-mdnplus"):
```

Comment:
Issue(blocking): If we're showing this for every Dev Edition WNP,  then this will almost certainly make the existing integration tests fail here:

- https://github.com/mozilla/bedrock/blob/main/tests/functional/firefox/whatsnew/test_whatsnew_developer_70.py
- https://github.com/mozilla/bedrock/blob/main/tests/functional/newsletter/test_newsletter_embed.py#L25
- https://github.com/mozilla/bedrock/blob/main/tests/functional/newsletter/test_newsletter_embed.py#L49
- https://github.com/mozilla/bedrock/blob/main/tests/functional/newsletter/test_newsletter_embed.py#L75

As long as this new template is behind a switch, then we can't easily add tests for the new page I think. So the options are:

- Disable the existing tests (not great, but OK for a limited time)
- Drop the switch and use version numbers instead (better coverage as we could have tests for both, but less flexibility without the switch).
- Keep the switch but continue to show the old template below a specific version number e.g. v70. (this might be the best option, as we can just ship the new page as-is but retain tests for the old page?).